src/components/Pill.tsx

```diff
@@ -1,4 +1,4 @@
-import { CloseIcon } from "./svg/Icons";
+import { Icon } from "./icon/Icon";
```

Comment:
tangent: where do we use relative vs aliased import paths?

Reply:
Aliased paths are used outside of the aliased directory. In this case you're seeing a relative path because it's one component importing another, rather than an unrelated file/page importing a component.